### ISSUE TITLE
Warningに対応する

### ISSUE THREAD

**Issue by kagomen** [kagomen/libraku#128]
現状発生しているワーニングは以下の通り

```
validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
```

```
`DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.
If you want to hide the `DialogTitle`, you can wrap it with our VisuallyHidden component.
For more information, see https://radix-ui.com/primitives/docs/components/dialog
```

```
Missing `Description` or `aria-describedby={undefined}` for {DialogContent}. 
```
